Library/Homebrew/test/utils/github_spec.rb

```diff
@@ -31,9 +31,12 @@
 
   describe "::search_issues", :needs_network do
     it "queries GitHub issues with the passed parameters" do
-      results = subject.search_issues("brew search", repo: "Homebrew/brew", author: "avetamine", is: "closed")
+      results = subject.search_issues("brew search",
+                                      repo: "Homebrew/legacy-homebrew",
+                                      author: "MikeMcQuaid",
+                                      is: "closed")
       expect(results).not_to be_empty
```

Comment:
Could actually remove this now, because the other test implies that it isn't empty.

Reply:
If it is empty it'll be a less pleasant error